scripting/2.0/modules/ROOT/pages/edits/scripting-module-upgrade-migrate.adoc

```diff
@@ -0,0 +1,109 @@
+= Upgrade Scripting Module to Version 2.0 - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Upgrade Scripting Module to version 2.0.0.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|1.1.7 |2.0.0
+|===
+
+== Changes in This Release
+
+This release contains the following changes:
+
+* Scripting Module no longer provides default scripting engines:
+** Use the *Configure...* helper button to install your desired engines.
+** Specify the desired engine in the `pom.xml` file of your application.
+* For any data stream provided in the payload or variables, a cursor object opens and binds, preventing the need for you to open new cursors. Any `openCursor()` provided to the Scripting Module closes after the script execution.
+
+
+== Upgrade Prerequisites
+
+Before you perform the upgrade, you must complete the following:
+
+* Create a backup of your files, data, and configurations in case you need to restore to them.
```

Comment:
```suggestion
Before you perform the upgrade, you must create a backup of your files, data, and configurations in case you need to restore to them.
```